### ISSUE TITLE
Incorrect lemma for the short form of 'I am'

### ISSUE THREAD

**Issue by dtinel** [explosion/spaCy#9924]
In the text: `I'm Dino, the family dog`

I got this for the token `'m`:
```
{
   "norm_": "am"
   "lemma_": "'m"
}
```

I think the lemma is wrong and the result should have been:
```
{
   "norm_": "am"
   "lemma_": "be"
}
```

When I wrote my text like this: `I am Dino, the family dog`, the lemma was found correctly.

I'm using the `en_core_web_sm` package:
```
nlp = spacy.load("en_core_web_sm")
doc = nlp("I'm Dino, the family dog")
```

**Comment by adrianeboyd** [comment]
Hi, this should be fixed in model release v3.2.0+. You can check `spacy validate` to see which version you have installed.

**Comment by dtinel** [comment]
@adrianeboyd this is my output of the command:

```
================= Installed pipeline packages (spaCy v3.2.1) =================
ℹ spaCy installation:
/home/user/.local/lib/python3.6/site-packages/spacy

NAME             SPACY            VERSION                            
en_core_web_sm   >=3.2.0,<3.3.0   3.2.0   ✔
```

I have just installed the `en_core_web_trf` accuracy package, and now it seems to find the lemma correctly.
